toolkit/Makefile

```diff
@@ -12,14 +12,17 @@ toolkit_root := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
 SCRIPTS_DIR  ?= $(toolkit_root)/scripts
 
 # Build targets
+##help:var:CONFIG_FILE:<config_path>=Path to image configuration file to use.
 ifeq ($(origin CONFIG_FILE), undefined)
 CONFIG_FILE             = $(toolkit_root)/imageconfigs/core-efi.json
-$(warning CONFIG_FILE is undefined, defaulting to toolkit's core-efi.json.)
+#$(warning CONFIG_FILE is undefined, defaulting to toolkit's core-efi.json.)
```

Comment:
While I also want' to get rid of this warning, I think we shouldn't leave the comment behind. Also, I think @dmcilvaney had a separate change dedicated to this clean-up (I might be misremembering something here, though).

Reply:
Okay, fixed in latest commit.

Reply:
Yup, I have a PR that removes this nonsense finally. Probably one of the biggest sources of long builds for users. Will just default to "" going forward.